packages/auth/src/AuthService.ts

```diff
@@ -13,16 +13,14 @@ export class AuthService {
   private auth: KeycloakInstance;
   private internalConfig: any;
 
-  constructor(appConfig: AeroGearConfiguration) {
-    const configuration = new ConfigurationHelper(appConfig).getConfigByType(AuthService.TYPE);
-
+  constructor() {
```

Comment:
in other SDKs, we have the `MobileCore` instance passed in as part of the constructor. I am wondering if we should do the same. Makes it bit easier to test, and also make it obvious that the service modules do depends on the core modules.

Reply:
@wei-lee Basing on what was agreed on list there will be work done on Android and Xamarin to not expose core. I intentionally done this without it.